literature.bib

```diff
@@ -23597,6 +23816,16 @@ @inproceedings{FPG:ICSM03
   doi = {10.1109/ICSM.2003.1235403}
 }
 
+@inproceedings{AOH:ISSRE03,
+	author={Ammann, P. and Offutt, J. and Hong Huang},
+	booktitle=ISSRE, 
+	title={{Coverage Criteria for Logical Expressions}}, 
+	year=2003,
+	pages={99-107},
+	keywords={Logic testing;Automata;FAA;Software safety;Aerospace electronics;Software engineering;Software testing;Terminology;Formal specifications;Automatic testing},
```

Comment:
Change to colons to match other keywords entries?

Reply:
You mean commas?

Reply:
Oops yes.